core/src/main/scala/com/broadinstitute/dsp/models.scala

```diff
@@ -12,4 +13,14 @@ object CloudService {
 final case class Disk(id: Long, googleProject: GoogleProject, diskName: DiskName) {
   override def toString: String = s"${id}/${googleProject.value},${diskName.value}"
 }
+
+//init buckets are different than staging buckets because we store them with gs://[GcsBucketName]/
+case class InitBucketName(value: String) extends ValueObject with Product with Serializable {
+  val asGcsBucketName: GcsBucketName = GcsBucketName(value)
+}
+
+final case class InitBucketToRemove(googleProject: GoogleProject, bucket: Option[InitBucketName]) {
+  override def toString: String = s"${googleProject.value},${bucket.getOrElse("null")}"
```

Comment:
didn't realize initBuckets are optional 🤔

Reply:
Yeah, I'm not sure why, but there are some NULLs in the dev DB